CloudM.Exercise.WebApi/Controllers/PackageController.cs

```diff
@@ -17,6 +19,8 @@ public PackageController()
         [Route("{name}/{version}")]
         public async Task<PackageDependencies> Get(string name, string version)
         {
+            var dependencyTree = new Dictionary<string, dynamic>();
+
             using var httpClient = new HttpClient();
             var response = await httpClient.GetAsync($"https://registry.npmjs.org/{name}");
```

Comment:
It will be better if this url is stored in a constant and in the appsettings file